src/main/kotlin/sipura/alg/GraphProperty.kt

```diff
@@ -219,4 +219,58 @@ object GraphProperty {
         }
         return Pair(degeneracy, ordering)
     }
+
+    /**
+     * Calculates the splittance of the given graph [g].
+     *
+     * The splittance of [g] is the smallest number of edge deletions or edge insertions that are required to
+     * transform [g] into a split graph. That means the splittance of [g] is zero if and only if [g] is a split graph.
+     *
+     * Runtime: O(n + m)
+     * @return the splittance of [g].
+     */
+    fun <V> minSplittance(g: SimpleGraph<V>): Int {
+        if (g.n == 0) return 0 // special case because maxDegree() throws exception for size 0
+        val maxDegree = g.maxDegree()
+        var degreeSumTotal = 0
+        // order vertices by degree in linear time
+        val degreeOrdering = Array<LinkedList<V>>(maxDegree + 1) { LinkedList() }
```

Comment:
create new function for degreeOrdering?
Seems like something that can be reused well.

Reply:
I created an Issue for that: https://github.com/sipura/Sipura/issues/67